inventory/group_vars/k8s-cluster.yml

```diff
@@ -54,6 +54,11 @@ kube_users:
     pass: "{{kube_api_pwd}}"
     role: admin
 
+## It is possible to deactivate selected authentication methods (basic auth,x509,token)
+#kube_x509_auth: false
```

Comment:
Disabling cert based auth impossible. kubelet uses it. Definitely get rid of this line.